sidebar-templates/sidebar-footerfull.php

```diff
@@ -16,7 +16,7 @@
 
 	<!-- ******************* The Footer Full-width Widget Area ******************* -->
 
-	<div class="wrapper" id="wrapper-footer-full" role="footer">
+	<div class="wrapper" id="wrapper-footer-full" role="complementary">
```

Comment:
```suggestion
	<aside class="wrapper" id="wrapper-footer-full">
```
And 
```diff
-</div><!-- #wrapper-footer-full -->
+</aside><!-- #wrapper-footer-full -->
```
on [line 31](https://github.com/understrap/understrap/blob/7b7fe079c0d60873c980aa67bf12e88105a49f81/sidebar-templates/sidebar-footerfull.php#L31)

Reply:
My concern here was about changing the semantic markup - I ran some tests and haven't seen any issues changing the `div` to an `aside` - how about you?

Reply:
Sorry, I was on a hurry, remembered the issue, saw your changes and didn't check your comment. You're right, a markup change should not be put in a feature release. Maybe it's best to open a different PR with milestone 2.0 so we don't forget to change the `div` to an `aside`?